CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
+## 8.4.0
```

Comment:
We typically add to the changelog when releasing, I'd remove this 8.4.0 section :-)